script.js

```diff
@@ -243,8 +246,37 @@ function formLowerCaseValidate(submitForm) {
   }
 }
 
+function storageEmailData() {
+  const nameInput = document.getElementById('name').value;
+  const emailInput = document.getElementById('mail').value;
+  const msgInput = document.getElementById('msg').value;
+  const formInfo = {
+    name: nameInput,
+    email: emailInput,
+    message: msgInput,
+  };
+  localStorage.setItem('formInfo', JSON.stringify(formInfo));
+}
+
+function preFillFields() {
+  const storageForm = JSON.parse(localStorage.getItem('formInfo'));
+  if (storageForm) {
+    nameField.value = storageForm.name;
+    mailField.value = storageForm.email;
+    messageField.value = storageForm.message;
+  }
+}
```

Comment:
* Kindly implement these functionalities as it is required
* The user-entered value be saved in local storage without clicking the submit button
* When the user changes the content of any input field, the data is saved to the local storage 
* Kindly 🙏 fix this as currently, the user has to click submit for input data to be stored in local storage and when changes are made on the input values the local storage isn't updated. [the specific Project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md#:~:text=When%20the%20user%20changes%20the%20content%20of%20any%20input%20field%2C%20the%20data%20is%20saved%20to%20the%20local%20storage.)